reg/drone/phy/kinematics/trans/LinearCovTs.0.1.uavcan

```diff
@@ -0,0 +1,10 @@
+# This is a structural subtype of LinearTs.
+# Use best guess if the error variance is unknown.
+
+LinearTs.0.1 value
+
+float16 position_error_variance         # [meter^2]
```

Comment:
how exactly is this just a `float16? You have position variance in x, in z and in z.

Reply:
This is a one-dimensional state.

Reply:
I see now the point. But as we discussed, it doesn't make sense to create a data type that fits one dimension and propagate a variance for one dimension and call it `position_error_variance`. What is conventional (you can check ROS or Mavlink as an example), is that you create a message/data type which is associated with linear/angular position/velocity/acceleration, and those are are 3 dimensional (points, vectors), or 4D if they are stamped.